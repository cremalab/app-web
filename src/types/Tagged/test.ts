import { Tagged } from "."

describe("tagged", () => {
  it("returns a `Tagged<Tag, Data>`", () => {
    // Arrange
    const tag = "SOME_TAG"
    const data = { id: "2", name: "Jeff" }

    // Act
    const received = Tagged(tag, data)

    // Assert
    expect(received).toMatchSnapshot()
  })
})
