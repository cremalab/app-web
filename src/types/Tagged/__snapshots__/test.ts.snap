// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tagged returns a \`Tagged<Tag, Data>\` 1`] = `
Object {
  "data": Object {
    "id": "2",
    "name": "Jeff",
  },
  "tag": "SOME_TAG",
}
`;
